Library/Formula/google-perftools.rb

```diff
@@ -1,9 +1,7 @@
-require 'formula'
-
 class GooglePerftools < Formula
-  homepage 'http://code.google.com/p/gperftools/'
-  url "https://googledrive.com/host/0B6NtGsLhIcf7MWxMMF9JdTN3UVk/gperftools-2.3.tar.gz"
-  sha1 "3f7d48a8dfd519f744d94cd2dc6a7875e456e632"
+  homepage "http://code.google.com/p/gperftools/"
```

Comment:
You should use `https` here. The build [is failing](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/19857/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_google_perftools/) because of that:

> Google Code homepages should be https:// links (URL is http://code.google.com/p/gperftools/)